field_audit/migrations/0002_remove_auditevent_field_audit_auditevent_valid_create_or_delete_and_more.py

```diff
@@ -0,0 +1,35 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I'm having trouble following the name of this migration file, and am wondering if there is a name that could better convey the intention of this migration. Totally fine if not.

Reply:
Good catch, this name totally slipped my mental review somehow (must have had tunnel vision on the `0002_` part, I guess?)

Reply:
Fixed in b184fc0.